src/index.html

```diff
@@ -3,9 +3,25 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Webpack Exercise</title>
+    <title>Book App</title>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <header class="header-section">
+        <div class="appName">
+            <h1 class="appTitle">Open Library Books</h1>
+        </div>
+
+        <div class="botHeader">
+          <div class="logoContainer">
+          </div>
+
+          <ul class="menuItems">
+            <li>Book</li>
```

Comment:
Very good job - nice and clean.
I have just a suggestion: Please, update the link to Books, not Book!